### ISSUE TITLE
datacarpentry/spreadsheet-ecology-lesson

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#40]
tracking issues for <https://github.com/datacarpentry/spreadsheet-ecology-lesson>

preview: https://fishtree-attempt.github.io/spreadsheet-ecology-lesson

- [x] auto-switch reference link anchors to use `-` and not `_` (this is causing the two "missing anchor" errors below to appear)
- [x] fix guide/ -> `instructors/instructor-notes.md`
- [x] glossary issue: `{: auto_id}` tag needs to disappear
- [x] figure out if fig/sorting_example.png actually exists (it does not)
- [x] figure out how to fix `! Could not process '/home/zhian/Documents/Carpentries/Git/carpentries/lesson-transition/datacarpentry/spreadsheet-ecology-lesson/setup.md': C++ exception: Extra content at the end of the document [5]` (to be fixed in https://github.com/datacarpentry/spreadsheet-ecology-lesson/pull/330)
- [ ] rejoin alt text attributes with images (see https://github.com/carpentries/pegboard/issues/119) [note: I will not fix this. The alt text here is almost worse than no alt text and hopefully it will help draw attention to the lack of good alt text]

```markdown
  episodes/01-format-data.md:143 [uninformative link text]: [here](https://ndownloader.figshare.com/files/2252083)
  episodes/02-common-mistakes.md:31 [missing anchor]: [Using formatting to make the data sheet look pretty](#formatting_pretty)
  episodes/02-common-mistakes.md:34 [missing anchor]: [Using problematic field names](#field_name)
  episodes/04-quality-control.md:134 [missing file]: [](fig/sorting_example.png)
  index.md:47 [missing file]: [Setup](setup.md)
  index.md:53 [missing file]: [Setup](setup.md)
  index.md:65 [missing file]: [Instructor notes](guide/)
  learners/setup.md:10 [missing file]: [data zip file](data/data.zip)
```

**Comment by zkamvar** [comment]
preview at https://fishtree-attempt.github.io/spreadsheet-ecology-lesson/

**Comment by zkamvar** [comment]
fixed in f814eb2704dc4dc755e8d9a2540c91e80e1cab5b
